samples/asgardeo-react-app/src/pages/home.tsx

```diff
@@ -73,6 +75,18 @@ export const HomePage: FunctionComponent<HomePagePropsInterface> = (
         })();
     }, [ state.isAuthenticated ]);
 
+    /* 
+    *   handles the error occurs when the logout consent page is enabled
+    *   and the user clicks 'NO' at the logout consent page
+    */
```

Comment:
Shall we format the comment?

Reply:
Fixes added with https://github.com/asgardeo/asgardeo-auth-react-sdk/pull/102